packages/pn-personafisica-webapp/src/pages/__test__/NotificationDetail.page.test.tsx

```diff
@@ -552,18 +552,25 @@ describe('NotificationDetail Page', () => {
         },
       });
     });
+
     const payButton = result?.getByTestId('pay-button');
-    const item = result?.getAllByTestId('pagopa-item')[requiredPaymentIndex];
-    const radioButton = item?.querySelector('[data-testid="radio-button"] input');
-    fireEvent.click(radioButton!);
+
     await waitFor(() => {
```

Comment:
rewrite the waitFor in this way
`const item = await waitFor(() => result?.queryAllByTestId('pagopa-item')[requiredPaymentIndex];)`
and put the other code outside the waitFor